test/e2e/specs/editor/blocks/buttons.spec.js

```diff
@@ -150,28 +150,6 @@ test.describe( 'Buttons', () => {
 		);
 	} );
 
-	test( 'can resize width', async ( { editor, page } ) => {
-		await editor.insertBlock( { name: 'core/buttons' } );
-		await page.keyboard.type( 'Content' );
-		await editor.openDocumentSettingsSidebar();
-		await page.click(
-			`role=region[name="Editor settings"i] >> role=tab[name="Settings"i]`
-		);
-		await page.click(
-			'role=group[name="Button width"i] >> role=button[name="25%"i]'
-		);
-
-		// Check the content.
-		const content = await editor.getEditedPostContent();
-		expect( content ).toBe(
-			`<!-- wp:buttons -->
-<div class="wp-block-buttons"><!-- wp:button {"width":25} -->
-<div class="wp-block-button has-custom-width wp-block-button__width-25"><a class="wp-block-button__link wp-element-button">Content</a></div>
-<!-- /wp:button --></div>
-<!-- /wp:buttons -->`
-		);
-	} );
-
```

Comment:
> Remove e2e tests that reference the scale button that was removed

This test, being for the Buttons block, seems to have been removed in haste. Excuse me if I am missing something though.

Reply:
My bad. I searched for `role=button[name="25%"i]` that was removed in the image block and mistakenly removed it here. Opened https://github.com/WordPress/gutenberg/pull/51865 to restore it.